### PR TITLE
Document snapshot generation process

### PR DESCRIPTION
**Pull request by lawrencejones**
As people have struggled to discover this otherwise!

### CODE REVIEW

File: integration/testdata/sync/actions.json [hunk 1]
```diff
@@ -9,4 +9,4 @@
     "status": "outstanding",
     "updated_at": "2023-10-24T15:10:57.97Z"
   }
-]
+]
```
Comment (lawrencejones) [inline integration/testdata/sync/actions.json:12]: This came from me running the update.

**Review by aaronsheah — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,6 +28,25 @@ Once installed, see [documentation](docs) for example configurations.
...
+incident.io account. These snapshots are not intended to be managed by hand, and
+should be updated automatically instead.
+
+Whenever you need to update the snapshots, run:
+
+```console
+$ export TEST_INCIDENT_API_KEY="<test-key>"
+$ export TAP_SNAPSHOT_UPDATE='true' ginkgo -tags=integration -r ./integration
+Writing snapshot file testdata/sync/follow_ups.json
+...
```
Comment (aaronsheah) [inline README.md:43]:
```suggestion
```

Comment (lawrencejones) [reply README.md:43]: I included this as an example of what you're get if you've done it right, would you advise not doing that?

Comment (aaronsheah) [reply README.md:43]: fine with that! thought it might be confusing to copy paste it but thats a small nit

Comment (lawrencejones) [reply README.md:43]: Ah so with the `console` type I'd normally signify actual commands with a `$` prefix while output lacks it. Helps avoid people copy and pasting a command and accidentally running it with the `$` prefix too!

Comment (aaronsheah) [reply README.md:43]: TIL!

**Pull request merged**